pkg/apiserver/internal/util/util.go

```diff
@@ -28,3 +34,38 @@ func TimestampProto(t *metav1.Time) (*timestamp.Timestamp, error) {
 	}
 	return ptypes.TimestampProto(t.Time)
 }
+
+func ValidateGetRequest(req *v1.GetRequest) error {
```

Comment:
I thought we want validation to be versioned.
maybe `pkg/apiserver/internal/v1/validation.go` is a better place for that, what do you think?

Reply:
Yeah, agree, that makes more sense, will do that

Reply:
Done